src/util/time_tricks.rs

```diff
@@ -30,6 +30,13 @@ static BEGINNING_OF_TIME: Lazy<(Instant, SystemTime)> = Lazy::new(|| {
     (beginning_of_time, beginning_of_time_sys)
 });
 
+fn beginning_to_epoch() -> Duration {
```

Comment:
Since the beginning is after the epoch I'd expect a negative value here, isn't this `epoch_to_beginning` i.e. the duration from the **epoch** _to_ the **beginning**

Reply:
Yep